tests/omni_lock_rust/tests/test_opentx.rs

```diff
@@ -0,0 +1,237 @@
+#![allow(dead_code)]
+mod misc;
+
+use ckb_script::TransactionScriptsVerifier;
+use ckb_types::packed::Byte32;
+
+use misc::*;
+use omni_lock_test::ckb_sys_call::{sys_call_dump_all, CkbSysCall};
+use omni_lock_test::dummy_data_loader::DummyDataLoader;
+
+fn debug_printer_d(_: &Byte32, msg: &str) {
+    print!("{}", msg);
+}
+
+//#[test]
+fn _test_dump_data() {
+    let mut data_loader = DummyDataLoader::new();
+
+    let mut config = TestConfig::new(IDENTITY_FLAGS_MULTISIG, true);
+    config.set_multisig(0, 2, 3);
+    config.smt_in_input = true;
+    //config.rep_dump_data = true;
+    config.scheme = TestScheme::OnWhiteList;
+
+    let lock_args = config.gen_args();
+    let tx = gen_tx_with_grouped_args(&mut data_loader, vec![(lock_args, 3)], &mut config);
+    let tx = sign_tx(&mut data_loader, tx, &mut config);
+    let resolved_tx = build_resolved_tx(&data_loader, &tx);
+
+    sys_call_dump_all(CkbSysCall::new(&resolved_tx.transaction, &data_loader));
+
+    let consensus = gen_consensus();
+    let tx_env = gen_tx_env();
+    let mut verifier =
+        TransactionScriptsVerifier::new(&resolved_tx, &consensus, &data_loader, &tx_env);
+    verifier.set_debug_printer(debug_printer_d);
+
+    let verify_result = verifier.verify(MAX_CYCLES);
+    verify_result.expect("pass verification");
+}
+
+fn gen_opentx_si_all() -> OpentxWitness {
```

Comment:
With only this one, it's not enough. Some other typical command combinations should be added.
You can always test them with "IDENTITY_FLAGS_PUBKEY_HASH".